RHEL/7/input/remediations/bash/accounts_tmout.sh

```diff
@@ -0,0 +1,7 @@
+# platform = Red Hat Enterprise Linux 7
+#. /usr/share/scap-security-guide/remediation_functions
+#popu#late var_#accounts_#tmout
+
+#echo "TMOUT=$var_accounts_tmout" >> /etc/profile
+
+echo "TMOUT=600" >> /etc/profile
```

Comment:
This line shouldn't exist due to the commented out line from the above. Also this script is not idempotent.